### ISSUE TITLE
Set group popup not opening

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#8113]
### Bug description

When you want to edit a set group to set an icon or a title, the action sheet to edit or delete a group of sets doesn't open anymore. I click the arrow and nothing happens.
<img width="630" alt="Screenshot 2023-05-11 at 10 26 53" src="https://github.com/statamic/cms/assets/69107412/6d186940-dacf-43b6-944e-de66eb962ada">

### How to reproduce

Add groups to a Replicator or Bard
Try to edit a group

### Environment

```yaml
Environment
Application Name: Statamic Peak Development
Laravel Version: 9.52.7
PHP Version: 8.2.3
Composer Version: 2.2.7
Environment: local
Debug Mode: ENABLED
URL: statamic-peak.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 4
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.0.0 PRO

Statamic Addons
studio1902/statamic-peak-browser-appearance: 2.0
studio1902/statamic-peak-commands: 2.0
studio1902/statamic-peak-seo: 2.0
studio1902/statamic-peak-tools: 3.0
```

### Installation

Starter Kit using via CLI

### Antlers Parser

None

**Comment by espenlg** [comment]
I have the same problem.

**Comment by jasonvarga** [comment]
Fixed by #8103
